foxy/distribution.yaml

```diff
@@ -4009,6 +4009,26 @@ repositories:
       url: https://github.com/rt-net/raspimouse_ros2_examples.git
       version: foxy-devel
     status: maintained
+  raspimouse_slam_navigation_ros2:
+    doc:
+      type: git
+      url: https://github.com/rt-net/raspimouse_slam_navigation_ros2.git
+      version: foxy-devel
+    release:
+      packages:
+      - raspimouse_navigation
+      - raspimouse_slam
+      - raspimouse_slam_navigation
+      tags:
+        release: release/foxy/{package}/{version}
+      url: https://github.com/ros2-gbp/raspimouse_slam_navigation_ros2-release.git
+      version: 1.0.0-1
+    source:
+      test_pull_requests: true
```

Comment:
@shu-rt has this been [setup](http://wiki.ros.org/buildfarm/Pull%20request%20testing) accordingly?

Reply:
@hidmic Hi, I must've missed some processes. We usually don't setup the `test_pull_request`
I will add a commit that removes that line.